python3/httplib2/__init__.py

```diff
@@ -969,10 +974,10 @@ def connect(self):
                 if has_timeout(self.timeout):
                     sock.settimeout(self.timeout)
                 sock.connect((self.host, self.port))
-                self.sock = ssl_wrap_socket(
+                self.sock = _ssl_wrap_socket(
```

Comment:
We're creating context in `Connection.__init__` only to forget it and create again in `_ssl_wrap_socket()`.

Intuitively, there should be only one ssl helper, something is wrong if we have two.

```python
  def connect(self):
    # ...
    self.sock = self._context.wrap...
```

Reply:
I think you are right here, there is a way to re-use the context in connect() instead of creating a new one. I will look into it.